docs_user/assemblies/assembly_adopting-the-shared-file-systems-service.adoc

```diff
@@ -4,21 +4,19 @@
 
 = Adopting the {rhos_component_storage_file}
 
-The {rhos_component_storage_file_first_ref} provides {rhos_prev_long} ({OpenStackShort})
-users with a self-service API to create and manage file shares. File
-shares (or simply, "shares"), are built for concurrent read/write access by
-any number of clients. This, coupled with the inherent elasticity of the
-underlying storage makes the {rhos_component_storage_file} essential in
-cloud environments with require RWX ("read write many") persistent storage.
+The {rhos_component_storage_file_first_ref} in {rhos_long} provides a self-service API to create and manage file shares. File shares (or "shares"), are built for concurrent read/write access by any number of clients. This makes the {rhos_component_storage_file} essential in cloud environments that require a ReadWriteMany persistent storage.
 
-File shares in {OpenStackShort} are accessed directly over a network. Hence, it is essential to plan the networking of the cloud to create a successful and sustainable orchestration layer for shared file systems.
+You access file shares in {rhos_acro} directly over a network. You must ensure that the networking in the {OpenStackShort} {rhos_prev_ver} environment matches the network plans for your new cloud after adoption. This ensures that tenant workloads remain connected to storage during the adoption process, even if the control plane has a minor interruption. The {rhos_component_storage_file} control plane services are not in the data path. Shutting down the API, scheduler, and share manager services do not impact access to existing shared file systems.
```

Comment:
{rhos_long} = Red Hat OpenStack Services on OpenShift 
{rhos_acro} = RHOSO
{OpenStackShort} = RHOSP

Are the sentences containing these attributes correct? I want to be sure I'm referring to the correct environment.